pkg/controller/remoteingress/remoteingress_controller_test.go

```diff
@@ -362,6 +364,26 @@ func TestRemoteClusterIngressReconcile(t *testing.T) {
 				fmt.Sprintf("%s-secondCertBundle-secret", testClusterName),
 			},
 		},
+		{
+			name: "Test setting httpErrorCodePages",
+			localObjects: func() []runtime.Object {
+				objects := []runtime.Object{}
+				cd := addIngressToClusterDeployment(testClusterDeployment(), "secondingress", "moreingress.example.com", nil, nil, "", configv1.ConfigMapNameReference{})
```

Comment:
I suspect this was meant to be
```suggestion
				cd := addIngressToClusterDeployment(testClusterDeployment(), "secondingress", "moreingress.example.com", nil, nil, "", testHttpErrorCodePages())
```